catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -24,6 +24,19 @@
     <bean id="attributeGroupMetacardType"
           class="org.codice.ddf.catalog.ui.forms.data.AttributeGroupType"/>
 
+    <bean id="shareableMetacard" class="org.codice.ddf.catalog.ui.metacard.sharing.ShareableMetacardImpl">
+        <property name="shareableMetacardType" ref="shareableMetacardType"/>
+    </bean>
```

Comment:
❗️ You can get rid of this `bean` and the metacard type `bean` since nobody is using them. They are just hanging out in the blueprint container and not being injected anywhere.